securedrop/sass/_source_index.sass

```diff
@@ -31,22 +36,55 @@
 
     .index-column
       padding: 1em
-      display: block
+      display: flex
+      flex-direction: column
+      justify-content: space-between
       width: 48%
-      border: 1px solid $color_purple_medium
       border-top-width: 0
       border-bottom-width: 0
 
     .index-left
-      background: $color_purple_medium
+      background-image: linear-gradient(140deg, #0096dc, #005db7)
       color: white
       border-style: none
 
-    .index-top
-      border-top-width: 1px
+    .index-right
+      background: white
+      border-width: 1.5px
+      border-image-source: linear-gradient(145deg, #0096dc, #005db7)
+      border-image-slice: 1
+      border: 1px solid
```

Comment:
I think you need to remove `border: 1px solid` here so it doesn't override the gradient border.

Reply:
Good catch, thanks! (Should be fixed.)